docs/source/reference/cpp.rst

```diff
@@ -67,13 +69,44 @@ below, excerpted from :file:`tests/test_calculator.cc`:
 .. literalinclude:: ../../../tests/test_calculator.cc
    :language: cpp
    :lines: 430-436, 441-444
-   :dedent: 2
+   :dedent: 4
+
+where :cpp:var:`representations.back()` is a
+:cpp:class:`RepresentationCalculator`,
+:cpp:var:`manager` is a :cpp:class:`StructureManager`, and
+:cpp:var:`structures.back()` is the :cpp:class:`AtomicStructure` associated with
+that manager.
+
+A more detailed documentation of these two classes follows:
+
+.. doxygenclass:: rascal::RepresentationManagerGradientCalculator
```

Comment:
small question: should these live in the rascal::test namespace?

Reply:
Didn't know that existed. Maybe, but that would require wrapping basically the whole file (`test_calculator.{cc,hh}`, as well as `test_math.hh` and everything it references) in a `namespace math` declaration.

Reply:
> Didn't know that existed.

It does not, the question was about creating it. And yeah on the wrapping, althought we could `using namespace rascal::test; using namespace rascal::math;` in the .cc file.